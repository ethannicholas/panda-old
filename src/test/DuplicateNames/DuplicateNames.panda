package this.is.awful

class String {
}

package something

class StringGetter {
    @class
    function getString():Class {
        return String
    }
}

package yet.another

uses this.is.awful.String

class StringGetter {
    @class
    function getString():Class {
        return String
    }
}

class Test {
    @class
    method main() {
        Console.writeLine(something.StringGetter.getString)
        Console.writeLine(yet.another.StringGetter.getString)
    }
}