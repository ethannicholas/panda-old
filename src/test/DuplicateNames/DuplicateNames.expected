panda.core.String
this.is.awful.String
